database/buildSchema/44_data_display.sql

```diff
@@ -29,3 +29,12 @@ CREATE TABLE data_display_column_definition (
     UNIQUE (table_name, column_name)
 );
 
+-- Table for cataloguing all "data" tables, including lookup tables
+CREATE TABLE data_table (
+    id serial PRIMARY KEY,
+    table_name varchar NOT NULL UNIQUE,
+    name varchar,
+    field_map jsonb,
```

Comment:
I'm not sure if we need "field_map", but I've kept it here for now. We can remove it later if we need to. It's not used for custom data tables, and I'm not sure how it's used in lookup tables, if at all.

Reply:
Okay, it's used in the front-end to build the dynamic query for querying the lookup tables. So it's fine, we don't need to use it for regular "data" tables, just lookup tables.